README.md

```diff
@@ -19,7 +19,7 @@ v2rayA mainly provides the following methods of installation:
 2. Docker
 3. Binary file and installation package
 
-See [**v2rayA - Wiki**](https://github.com/v2rayA/v2rayA/wiki/Usage)
+See [**v2rayA - Wiki**](https://v2raya.org/en/docs/prologue/introduction/)
```

Comment:
Can you modify the title to v2rayA - Docs?

Reply:
Fixed